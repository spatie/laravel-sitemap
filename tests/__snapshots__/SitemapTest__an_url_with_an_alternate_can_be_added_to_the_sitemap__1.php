<?php

return '<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
        http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
    <url>
    
    <loc>http://localhost/home</loc>
        <xhtml:link rel="alternate" hreflang="nl" href="http://localhost/thuis" />
        <xhtml:link rel="alternate" hreflang="fr" href="http://localhost/maison" />
        <lastmod>2016-01-01T00:00:00+00:00</lastmod>
        <changefreq>daily</changefreq>
        <priority>0.8</priority>
    </url>
</urlset>
';
