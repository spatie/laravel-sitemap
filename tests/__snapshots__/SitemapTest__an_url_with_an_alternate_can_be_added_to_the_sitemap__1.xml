<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
  <url>
    <loc>http://localhost/home</loc>
    <xhtml:link rel="alternate" hreflang="nl" href="http://localhost/thuis"/>
    <xhtml:link rel="alternate" hreflang="fr" href="http://localhost/maison"/>
    <lastmod>2016-01-01T00:00:00+00:00</lastmod>
    <changefreq>daily</changefreq>
    <priority>0.8</priority>
  </url>
</urlset>
